### PR TITLE
MAGE-957: js and css reorged.

### PR DESCRIPTION
**Pull request by mrahman3177**
js and css reorganized.

### CODE REVIEW

**Review by damcou — CHANGES_REQUESTED** [review]
Looks fine !
Maybe we should put the `autocomplete-plugin-recent-searches.js` file in an `internals` folder as we did in the extension ? WDYT ?

**Comment by mrahman3177** [comment]
That's a good idea to be consistent. Thanks for pointing it out.

**Review by damcou — APPROVED** [review]
LGTM

**Review by cammonro — CHANGES_REQUESTED** [review]
@mrahman3177 Let's bump this to version 1.3 and tag it with a `^3.14.0` dependency on the main module `algolia/algoliasearch-magento-2`.

File: composer.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   ],
   "version": "1.2.2",
```
Comment (cammonro) [inline composer.json]:
```suggestion
  "version": "1.3.0",
```

**Review by cammonro — CHANGES_REQUESTED** [review]
@mrahman3177  Let's be sure to bump the version whenever we merge to master. I'm thinking we should bump by a minor version for every minor version in the main extension. `1.3.0` since we just released `3.14.0`. WDYT?

**Review by cammonro — APPROVED** [review]

**Pull request merged**